integr8ly/installation
# mdc install improvements

Update the manifest template file so that it can be used by MDC. Also add user-sso as a mobile service

## Verification Steps

Run the installation playbook, or check this cluster: https://master.demo27-8353.openshiftworkshop.com, and the generated manifest file [here](https://master.demo27-8353.openshiftworkshop.com/console/project/mobile-developer-console/browse/secrets/manifest).
